### ISSUE TITLE
Random Encounter with Imperials Escorting Stormcloak Prisoner Not Moving

### ISSUE THREAD

**Issue by Eve-Knight** [schofida/CWORR#74]
On the way from Riverwood to Whiterun, I encountered the common group of three imperial soldiers escorting a bound Stormcloak prisoner. They were all just standing still rather than walking anywhere, though. They still told me I was interfering with their business and such when I got close.

Presuming nothing with them changed in the latest CWO version, this probably has something to do with JK's Whiterun Outskirts. I tried temporarily disabling it, but that didn't change their behavior. Maybe they work properly if the outskirts mod is not enabled when they first spawn. Maybe they're not connected at all.

**Comment by LABWAZHERE** [comment]
saw that too can confirm

**Comment by schofida** [comment]
CWO does not change any World events (like patrols and stuff) so I don't think it's this. Is this on a new game? If that's the case, I will see if I can find the same group and reproduce the issue..

**Comment by LABWAZHERE** [comment]
yes new game for me

**Comment by LABWAZHERE** [comment]
they looked into the forest for me however

**Comment by Eve-Knight** [comment]
Yes, new game here as well. My understanding is that CWO just alters the pool of random soldiers that can be escorting the prisoner so we can now get at least archers and wizards if not more (this time I just got three plain soldiers, though) ... so yeah, I don't see how that would break this either.

**Comment by schofida** [comment]
Fixed by latest USSEP updates :)
